pages/index.tsx

```diff
@@ -3,9 +3,20 @@ import {NextPage} from "next";
 import scss from "../pageStyles/index.module.scss"
 import Layout from "../components/Layout/layout";
 
-const Home: NextPage = () => 
+const Home: NextPage = () =>
     <Layout>
         <h1 data-testid="page-render-test" className={scss.example}>Hello Admin Site</h1>
     </Layout>;
 
+Home.getInitialProps = ({res}) => {
```

Comment:
`getInitialProps` are basically deprecated now for NextJS - https://nextjs.org/docs/api-reference/data-fetching/getInitialProps

we probably want 'getStaticProps' here instead?